### PR TITLE
[Feature] Add Transferal Perceptual Loss

### PR DESCRIPTION
**Pull request by Yshuo-Li**

### CODE REVIEW

File: mmedit/models/losses/perceptual_loss.py [hunk 1]
```diff
@@ -202,3 +206,60 @@ def _gram_mat(self, x):
         features_t = features.transpose(1, 2)
         gram = features.bmm(features_t) / (c * h * w)
         return gram
+
+
+@LOSSES.register_module()
+class TPerceptualLoss(nn.Module):
```
Comment (innerlee) [inline mmedit/models/losses/perceptual_loss.py:212]: A single `T` is hard to guess. Use the full name is ok

File: mmedit/models/losses/perceptual_loss.py [hunk 2]
```diff
@@ -209,19 +209,18 @@ def _gram_mat(self, x):
...
-class TPerceptualLoss(nn.Module):
-    """Texture perceptual loss.
+class TransferalPerceptualLoss(nn.Module):
+    """Transferal perceptual loss.
 
     Args:
         loss_weight (float): Loss weight. Default: 1.0.
-        use_s (bool): If True, use soft-attention tensor in TPerceptualLoss.
-            Default: True.
+        use_s (bool): If True, use soft-attention tensor. Default: True.
```
Comment (innerlee) [inline mmedit/models/losses/perceptual_loss.py:217]: Same here. `s` is very cryptic.

**Comment by innerlee** [comment]
On names. They should be accessible both for beginners who don't know what they are and for experts who knows a lot similar concepts. It is not for the author him/herself who knows exactly what a letter "s" or "T" stands for.

File: mmedit/models/losses/perceptual_loss.py [hunk 3]
```diff
@@ -202,3 +206,59 @@ def _gram_mat(self, x):
         features_t = features.transpose(1, 2)
         gram = features.bmm(features_t) / (c * h * w)
         return gram
+
+
+@LOSSES.register_module()
+class TransferalPerceptualLoss(nn.Module):
```
Comment (innerlee) [inline mmedit/models/losses/perceptual_loss.py:212]: Double check Texture vs Transferal. If the latter, pls modify the pr title accordingly

**Review by innerlee — APPROVED** [review]
